### DISCUSSION TITLE
Multi-pass swap?

### DISCUSSION THREAD

**Discussion by dragonpam** [kex0/batch-face-swap#28] [Ideas]
Feature request: If there's more than one face in a picture, do a multi-pass swap (different prompts) to replace with different personas, with some predictable heuristic (e.g. X faces, left to right, Face A and then Face B).

Unless this is somehow possible already, would be great for workflows in comics, etc.

And if it's possible already, could someone post an example of how this works?

**Comment by rubberbaron** [comment]
This is similar to #25

**Reply by dragonpam** [reply]
yes, it's similar - thank you. So I think if you have a simple heuristics for specifying a subject, then you're definitely on the right track for this request left to right request is order faces detected (1... n), and apply separate prompts to them.  there may be other heuristics rather than L-to-R, but i'll keep this request simple.  

thank you for considering it.
